include/libserver/network/command/proto/RanchMessageDefinitions.hpp

```diff
@@ -552,9 +553,9 @@ struct RanchCommandSearchStallionOK
     uint32_t tid{};
     std::string name{};
     uint8_t grade{};
-    uint8_t chance{};
+    uint8_t inheritanceRate{};
```

Comment:
Care to elaborate on this field with comments? Please don't do sideline comments, use proper doxygen documentation.

```cpp
//! This field does that
uint32_t field;
```